source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,865 @@
 .. _mixer:
 
-Use the mixer tool
-##################
+mixer
+#####
 
-*Mixing* refers to composing an operating system for specific use cases.
-While the default |CL-ATTR| provides options to install bundles for various
-server capabilities, some developers may wish to either augment the
-operating system itself with functionality from their own packages or modify
-the structure of current bundles to cater to their particular needs.
+mixer is the tool used by the |CL-ATTR| team to generate official update content
+and releases. The update content generated by mixer is then consumed by swupd on
+a downstream client. The same mixer tool is available as part of |CL| to create
+your own customized update content and releases.
 
-Prerequisites
-*************
+.. contents::
+   :local:
+   :depth: 1
 
-To start working with the mixer tools, you need a recent image of |CL| with
-the `mixer` bundle installed. If the bundle is not yet installed, you can
-add it with the :command:`swupd bundle-add` command as follows:
+Description
+***********
 
-.. code-block:: bash
+mixer uses the following sources as inputs to generate update content:
 
-   sudo swupd bundle-add mixer
+* Upstream |CL| update content
```

Comment:
I would suggest the following instead:
- Upstream |CL| bundles with their corresponding RPM packages
- Locally-defined bundles with their corresponding local RPM packages

Mixer doesn't take in RPMs as inputs.  It takes in bundle names which contains lists of RPMs.